docs/ref/filterset.txt

```diff
@@ -130,6 +130,26 @@ This is a map of model fields to filter classes with options::
 Overriding ``FilterSet`` methods
 --------------------------------
 
+When overriding classmethods, calling ``super(MyFilterSet, cls)`` may result
```

Comment:
These `super` calls don't look right.

Reply:
What doesn't look right here? The example is intended to show how you would accidentally generate a `NameError`.